### PR TITLE
Do not exit the script if can not find rpms

### PR DESCRIPTION
**Pull request by nilxam**
once rpms2solv fails to find rpms it will reiggered exiting the script, check it had rpms there before do rpms2solv, otherwise return back and go next project.

### CODE REVIEW

File: create_test_dvds.sh [hunk 1]
```diff
@@ -82,9 +82,15 @@ function sync_prj() {
...
-	    local start=$SECONDS
-	    rpms2solv $dir/*.rpm > $dir.solv
-	    echo "creating ${dir}.solv took $((SECONDS-$start))s"
+        rpms=$(ls $dir/*.rpm 2> /dev/null | wc -w)
+        if [ $rpms != 0 ]; then
+            local start=$SECONDS
+            rpms2solv $dir/*.rpm > $dir.solv
+            echo "creating ${dir}.solv took $((SECONDS-$start))s"
+        else
+            echo "can not found any rpm file in ${dir}"
```
Comment (DimStar77) [inline create_test_dvds.sh]: Cannot find any rpm file in ${dir}

**Comment by nilxam** [comment]
corrected. thanks for review!

File: create_test_dvds.sh [hunk 2]
```diff
@@ -82,9 +82,15 @@ function sync_prj() {
     mkdir -p $dir
     perl $SCRIPTDIR/bs_mirrorfull --nodebug https://build.opensuse.org/build/$prj/$arch $dir
     if [ "$dir" -nt "$dir.solv" ]; then
-	    local start=$SECONDS
-	    rpms2solv $dir/*.rpm > $dir.solv
-	    echo "creating ${dir}.solv took $((SECONDS-$start))s"
+        rpms=$(ls $dir/*.rpm 2> /dev/null | wc -w)
+        if [ $rpms != 0 ]; then
```
Comment (lnussel) [inline create_test_dvds.sh]: shopt -s nullglob
rpms=(*.rpm)
if [ "${#rpms}" -gt 0 ] ; then
...
rpm2solv "${rpms[@]}" ...

**Comment by lnussel** [comment]
you may want to put the shopt at the begining of the script though

**Pull request merged**